platforms/ios/framework/src/TGMapFeature+Internal.h

```diff
@@ -0,0 +1,40 @@
+//
+//  TGMapFeature+Internal.h
+//  tangram
+//
+//  Created by Matt Blair on 2/25/19.
+//
+
+#import "TGMapFeature.h"
+
+#pragma mark Common Map Feature Behavior
+
+@interface TGMapFeature ()
+
+- (instancetype)initWithProperties:(TGFeatureProperties *)properties;
```

Comment:
Whats the use case for a map feature without any associated geo location but just properties?

Reply:
Ok, I see its used internally to instantiate parent `TGMapFeature`: https://github.com/tangrams/tangram-es/blob/53b302197e0325f84b2f0525193c7ffde23942bb/platforms/ios/framework/src/TGMapFeature.m#L61